apps/events-helsinki/tsconfig.json

```diff
@@ -33,6 +33,7 @@
     "**/*.cjs",
     "**/*.mjs",
     "**/*.jsx",
-    "**/*.json"
+    "**/*.json",
+    "../hobbies-helsinki/src/domain/app/headlessCmsHelper.tsx"
```

Comment:
what is this?

Reply:
some reason why the vscode adds to automatically. It's an accident. Nice catch!